### PR TITLE
Review docs to appease RoboDog

### PR DESCRIPTION
**Pull request by issyl0**
- Here we go. Appease the RoboDog (the bot that posts in #govuk-developers every day that we haven't reviewed our docs).
- Some of these aren't even date bumps as I wanted to add more but didn't have enough 🧠. I've left those for another willing person. But I did do some rewording.
- The commits to review are the ones I've actually edited, so the ones that have more than a single line description.

----

- I hope you lot appreciate my dedication to commit messages. I realised half way through (when I got to `errors`) that I got the date wrong - I'd been thinking it was 17th not 18th - so I went back and rebase-edited all my commits before then to change each date. 😩
- I've also not done *all* the ones that RoboDog moaned about - but there are only eight left for YOU! 💪

### CODE REVIEW

File: source/manual/help-with-publishing-content.html.md [hunk 1]
```diff
@@ -4,61 +4,53 @@ title: Help with publishing content
...
+  If all looks fine, it may be necessary to [manually create a `ContentChange` in
+  Email Alert API](https://github.com/alphagov/email-alert-api/blob/1aee9703bf303d43ba4ecb5f6fd771b757d52daf/app/services/notification_handler_service.rb#L24-L43).
+
+  We have an alert that checks that email alerts for Travel Advice and Medical
+  Safety Alerts go out correctly. Although an alert won't be triggered for other
+  kinds of documents, the [guidance will still apply](alerts/email-alerts-travel-medical.html).
+
+- If [a scheduled publication hasn't gone live][scheduled-publishing],
+  you should start by following the instructions for [if documents
+  aren't live after being published][live]. If it looks as though the
```
Comment (reggieb) [inline source/manual/help-with-publishing-content.html.md:46]: I'd suggest this is reworded:
```md
- If [a scheduled publication hasn't gone live][scheduled-publishing],
  you should start by following the instructions: [if documents
  aren't live after being published][live].
```

Comment (issyl0) [reply source/manual/help-with-publishing-content.html.md:46]: You mean as a separate paragraph to the next sentence? OK.

Comment (issyl0) [reply source/manual/help-with-publishing-content.html.md:46]: Wait... sorry, I really can't see the problem here!

Comment (reggieb) [reply source/manual/help-with-publishing-content.html.md:46]: I think that "start by following the instructions for if documents aren't live after being published" doesn't flow very well. Accepting that the format may make it clear that "if documents" starts the description of the document being linked to, I just think the using a punctuation point like ":" would make that break more obvious. So the text reads:

> start by following the instructions: if documents aren't live after being published

Comment (issyl0) [reply source/manual/help-with-publishing-content.html.md]: I fixed this up. Thanks!

**Comment by issyl0** [comment]
This is a _fun_ test failure:

```
expected: > 2020-05-19 09:11:44.247229756 +0000
got:   2020-05-19 09:11:43.396111009 +0000
```

**Pull request merged**